modules/example-apache-httpd-configuration.adoc

```diff
@@ -44,7 +58,7 @@ scripting in Apache HTTPD to accomplish this. This section provides examples.
 #!/bin/bash
 echo "Content-Type: application/json"
 echo ""
-echo '{"sub":"userid", "name":"'$REMOTE_USER'"}'
+echo '{"<IDP_NAME>":"userid", "name":"'$REMOTE_USER'"}'
```

Comment:
> I will update the docs accordingly and include the fixes for ... userid as well

Hi, we need replace the hard code "userid" instead of change the key name "sub", correct fix is:
```
echo '{"sub":"'$REMOTE_USER'", "name":"'$REMOTE_USER'"}'
```

Reply:
Done!